### PR TITLE
[PROD-840] Correcting Gentle build

### PR DESCRIPTION
**Pull request by brian-dlee**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    gfortran \
+    libtool \
+    subversion \
+    g++
+
+ADD gentle/ext /gentle/ext
+WORKDIR /gentle/ext
+
+# Compile Kaldi using OpenBLAS
+ARG OPENBLAS_COMMIT=tags/v0.3.7
```
Comment (moizzd) [inline docker/Dockerfile]: Make the version an evn var. Then we can set it in the circleCi config file.

**Review by moizzd — APPROVED** [review]
lgtm

**Comment by moizzd** [comment]
But the kaldi build is still failing....

**Comment by brian-dlee** [comment]
There were some mistakes during cleanup. It's passing, but it takes almost an hour to build.

File: docker/Dockerfile (hunk 1 above)
Comment (brian-dlee) [reply docker/Dockerfile]: It is piped in from the ENV in the build.sh script

File: docker/build.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+cp -R ../gentle gentle/gentle
+cp ../align.py gentle/align.py
+cp ../setup.py gentle/setup.py
+cp ../serve.py gentle/serve.py
+cp ../install_models.sh gentle/
+cp ../install_language_model.sh gentle/
+
+if [[ -n "$MAKE_NUM_THREADS" ]]; then BUILD_ARGS="$BUILD_ARGS --build-arg MAKE_NUM_THREADS=$MAKE_NUM_THREADS"; fi
+if [[ -n "$OPENBLAS_NUM_THREADS" ]]; then BUILD_ARGS="$BUILD_ARGS --build-arg OPENBLAS_NUM_THREADS=$OPENBLAS_NUM_THREADS"; fi
+if [[ -n "$OPENBLAS_COMMIT" ]]; then BUILD_ARGS="$BUILD_ARGS --build-arg OPENBLAS_COMMIT=$OPENBLAS_COMMIT"; fi
```
Comment (brian-dlee) [inline docker/build.sh]: It's passed in here if it is set

**Pull request merged**